modules/menu/menu.c

```diff
@@ -662,7 +662,7 @@ struct ua   *menu_ua_carg(struct re_printf *pf, const struct cmd_arg *carg,
 	if (ua)
 		return ua;
 
-	err = re_regex(carg->prm, str_len(carg->prm), "[^ ]+ [^ ]+", word1,
+	err = re_regex(carg->prm, str_len(carg->prm), "[^ ]+ [0-9]+", word1,
```

Comment:
Looks good.

And maybe we should rename parameter `word2` to `uaid`. Or should we remove it? It is used only internally.

Reply:
Can we be sure that `word1` is set correctly if no second word is specified?